packages/ui-extensions/docs/surfaces/customer-account/reference/targets/customer-account.page.render.doc.ts

```diff
@@ -7,10 +7,17 @@ import {
 
 const data: ReferenceEntityTemplateSchema = {
   name: 'customer-account.page.render',
-  description: `This [full-page extension](/docs/api/customer-account-ui-extensions/extension-targets-overview#full-page-extension-target) allows you to create a new page in customer accounts. It renders in the main content area, below the header, and above the footer.`,
+  description: `This [full-page extension](/docs/api/customer-account-ui-extensions/unstable/extension-targets-overview#full-page-extension-full-page-extension) allows you to create a new page in customer accounts. It renders in the main content area, below the header, and above the footer.\n
```

Comment:
the only difference between code and Figma 
- Existing code: `It renders in the main content area, below the header, and above the footer.`
- Figma: `It renders in the main content area - below the header, and above the footer.` 

I will confirm with @deannatron  before merging

Reply:
updated, using em-dash (—) and no space

Reply:
<img width="685" alt="Screenshot 2024-09-30 at 11 06 59 AM" src="https://github.com/user-attachments/assets/4d34e46a-7666-49e1-aa62-f4c90ebafef6">